app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -2,7 +2,8 @@ child :content_facet => :content_facet_attributes do
   extends 'katello/api/v2/content_facet/base'
 
   node do |content_facet|
-    version = content_facet.content_view_version
+    content_facet.single_content_view # just so Rails gives a warning and we remember to fix this
```

Comment:
What's the warning here? Users are often sensitive to warnings/errors filling up log files, so I wonder if we could rely instead on a redmine issue and a FIXME comment.

Reply:
It's the warning elsewhere in this PR saying that `single_content_view` is deprecated.